File: backend/Multibanca.DTO/Multibanca/ActividadDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.DTO.Multibanca
{
    public class ActividadDTO
    {
        public long nro_row { get; set; }

        public long id_expediente { get; set; }

        public string? id_actividad { get; set; }

        public string? actividad { get; set; }

        public DateTime? fecha_asignacion { get; set; }

        public string? estado { get; set; }

        public string? rol { get; set; }

        public string? nombre_responsable { get; set; }

        public string? url_act { get; set; }

        // Extras
        public int id_etapa { get; set; }

        public string[]? title { get; set; }
    }
}

File: frontend/src/features/bandeja_actividades/models/ActividadDTO.ts
export interface ActividadDTO {
  nro_row: number;
  id_expediente: number;
  id_actividad?: string | null;
  actividad?: string | null;
  fecha_asignacion?: string | null;
  estado?: string | null;
  rol?: string | null;
  nombre_responsable?: string | null;
  url_act?: string | null;

  id_etapa?: number;
  title?: string[] | null;
}
